definitions/npm/styled-components_v2.x.x/flow_v0.53.x-/styled-components_v2.x.x.js

```diff
@@ -0,0 +1,332 @@
+// @flow
+
+type $npm$styledComponents$Interpolation = ((executionContext: Object) => string) | string | number;
+type $npm$styledComponents$NameGenerator = (hash: number) => string;
+
+type $npm$styledComponents$TaggedTemplateLiteral<R> = {| (Array<string>, $npm$styledComponents$Interpolation): R |};
+
+type $npm$styledComponents$ReactComponentFunctional<Props: {}> = Props => React$Element<*>;
+type $npm$styledComponents$ReactComponentClass<Props, DefaultProps = *> = Class<React$Component<DefaultProps & Props, *>>
+
+type $npm$styledComponents$ReactComponentUnion<Props> =
```

Comment:
What sort of issues?  I ran into some when I was working on bumping it locally, but they resolved once everything was switched over.